frontend/config.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
put this file to gitignore please and remove it here

Reply:
done